scripts/RUN_59_AFTER.sh

```
@@ -1,10 +1,11 @@
+#!/bin/bash
 # Run at :59 after the hour, some stuff to get a jump on the next hour
 cd ingestors/madis
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)